app/src/docs/v1.api-spec.yaml

```diff
@@ -18,26 +18,26 @@ security:
 paths:
   /health:
     get:
-      summary: (DRAFT) Returns status of correspondent APIs
+      summary: Returns status of external dependencies
```

Comment:
"external service dependencies" may be better - as dependencies alone could also imply code libraries.

Reply:
ok